README.md

```diff
@@ -1,6 +1,6 @@
 # IBM Spectrum Scale container native
 
-IBM Spectrum Scale in containers allows the deployment of the cluster file system in a Red Hat® OpenShift® cluster. Using a remote mount attached file system, the IBM Spectrum Scale solution provides a persistent data store to be accessed by the applications via the [IBM Spectrum Scale Container Storage Interface (CSI) driver](https://www.ibm.com/docs/en/spectrum-scale-csi) using Persistent Volumes (PVs).
+IBM Spectrum Scale in containers allows the deployment of the cluster file system in a Red Hat® OpenShift® cluster. Using a remote mount attached file system, the IBM Spectrum Scale solution provides a persistent data store to be accessed by the applications via the [IBM Spectrum Scale Container Storage Interface (CSI) driver](https://www.ibm.com/support/knowledgecenter/STXKQY_CSI_SHR/com.ibm.spectrum.scale.csi.v2r10.doc/bl1csi_kc_landing.html) using Persistent Volumes (PVs).
```

Comment:
we shouldn't point at old version of CSI documentation

Reply:
Oh I see that now…

Reply:
by the way I didn't see this when I did v5.1.2.1 last week... Laura pointed it out.  I forgot to update internal I bet.